### PR TITLE
Added support for SUSE, also added option to not incluse csp

### PR DESCRIPTION
**Pull request by dstam**

### CODE REVIEW

**Comment by basvandervlies** [comment]
Thanks a for the contribution. I will look into it today.

File: services/apache.cf [hunk 1]
```diff
@@ -2,31 +2,76 @@
...
+                        "php7.3-mysql": "",
+                        "php7.3-pgsql": "",
+                        "php7.3-ldap": "",
+                        "php7.3-xml": "",
+                        "php-xdebug": "",
+                        "php7.3-bcmath": "",
+                        "php7.3-imap": "",
+                        "php7.3-json": "",
+                        "php7.3-soap": ""
+                    }
```
Comment (basvandervlies) [inline services/apache.cf:47]: Leave out the `php` packages as before. This can be installed in a seperate `json` file if neeeded.

Comment (dstam) [reply services/apache.cf:47]: Woops, my mistake, removed the packages and pushed to mu branch.

**Comment by dstam** [comment]
I just noticed I missed to add some required files that are specific for suse.

**Pull request merged**